app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -81,8 +86,17 @@ protected static void createFirebaseDoneFlag()
     protected static void createFirebaseTestUserWithCredentials(String email, String pwd)
             throws ExecutionException, InterruptedException {
         Task<AuthResult> t = FirebaseAuth.getInstance().createUserWithEmailAndPassword(email, pwd);
-        FirebaseAuth.getInstance().signOut();
         Tasks.await(t);
+        FirebaseAuth.getInstance().signOut();
+    }
+
+    /**
+     * This method deletes a user, it is assumed the user is logged in.
+     */
+    protected static void deleteTestUserWithCredentials(String email, String password)
+            throws ExecutionException, InterruptedException {
+        Task<Void> t2 = Objects.requireNonNull(FirebaseAuth.getInstance().getCurrentUser()).delete();
+        Tasks.await(t2);
     }
```

Comment:
That's how I wrote it first, thats why there is the credentials I forgot to remove them. But I think its better this way since there is already a function to sign in just below and when deleteUser is used the user is most likely already signed in, if not the person can call signInWithCredentials first.

Reply:
Alright in this case maybe just remove the argument and rename the function to something like deleteSignedInUser